param/parameterized.py

```diff
@@ -29,6 +29,7 @@
 from itertools import chain
 from operator import itemgetter, attrgetter
 from types import FunctionType, MethodType
+from typing import Any, Union, Literal, Iterable, Callable # When python 3.9 support is dropped replace Union with |
```

Comment:
I learned that `typing.Callable` and `typing.Iterable` are sort of deprecated, instead one should import them from `collections.abc`. See https://docs.python.org/3/library/typing.html#typing.Iterable.

Reply:
Updated.